devenv/datasources.yaml

```diff
@@ -255,6 +255,7 @@ datasources:
       customMetricsNamespaces: "CWAgent"
 
   - name: gdev-loki
+    uid: gdev-loki
```

Comment:
@grafana/observability-logs: Could this be problematic in development setup? Without providing the `uid` the data source gets randomly generated one.